lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/qm/XwikiCoreFacadeRestResource.java

```diff
@@ -92,8 +94,7 @@ public void genrationCompleted(String questionnairesId)
 		String uri = String.format("%s/learnpad/qm/corefacade/genrationcompleted/%s",
 				RestResource.REST_URI, questionnairesId);
 		PutMethod putMethod = new PutMethod(uri);
-		putMethod.addRequestHeader("Accept", "application/xml");
-
+		putMethod.addRequestHeader("Accept", MediaType.TEXT_PLAIN);
```

Comment:
Not using an XML structure for your questionnaire?  `text/plain` doesn't make much sense because questionnaire is a structured information so I would have understand any of `application/xml`, `application/json` or HTML but I don't understand `text/plain`.  If it's temporary (because simpler for now) and then it'll evolve, then I'm fine to merge it.